ml-dsa/src/lib.rs

```diff
@@ -913,8 +927,7 @@ pub trait KeyGen: MlDsaParams {
     fn key_gen<R: CryptoRng + ?Sized>(rng: &mut R) -> Self::KeyPair;
 
     /// Deterministically generate a signing key pair from the specified seed
-    // TODO(RLB): Only expose this based on a feature.
```

Comment:
I am not sure of this comment in particular. There may be reasons `key_gen_internal` cannot be named `from_seed` that I'm not aware of, and I can't think of reasons for feature gating this? cc. @tarcieri @bifurcation

Reply:
The name comes from [FIPS 204](https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.204.pdf), namely section 6.1 where it is referred to as `ML-DSA.KeyGen_internal`.

I don't oppose the change in this PR, though, just providing a reference for why it's named the way it is.

Reply:
Thanks, I've updated the docstrings to include that for end users.